### DISCUSSION TITLE
Student Developer Pack - long waiting time

### DISCUSSION THREAD

**Discussion by fcholak** [community/community#80897] [GitHub Education]
### Hi everyone,

When is my application getting processed? I've been waiting for 5 days already. 

Best regards,

Filip

**Comment by samus-aran** [comment]
Hi @fcholak 👋 

Please check the status of your application [here](https://education.github.com/discount_requests/pack_application). If your application is still pending review then it will appear at the top of the sidebar to the right of the page. If your application has already been approved then you will be redirected to your GitHub Education benefits.

What we can see is that your latest application is still in "pending" which would imply that you are still required to provide additional information. Once you have done this you can submit and it can only then be processed to be approved. Hope that helps.
